### PR TITLE
Updated the default path for logs in KSQL.

### PR DESCRIPTION
**Pull request by hjafarpour**
This PR will set the default path for the logs in `/tmp/ksql-logs` folder. This includes both server side logs and cli ones.
This solves #855

### CODE REVIEW

**Comment by apurvam** [comment]
Thanks for the patch @hjafarpour . How will a user be able to configure the log directory to be something other than `/tmp`? In most operating systems, `/tmp/` gets cleared periodically, so it is not an ideal place for logs of a long running service.

**Comment by hjafarpour** [comment]
@apurvam the log path can be set in the corresponding log properties files. This is just a default value and can be changed by user if desired.

File: config/log4j.properties [hunk 1]
```diff
@@ -1,5 +1,7 @@
 log4j.rootLogger=INFO, stdout
 
+ksql.log.dir=/tmp/ksql-logs/
```
Comment (kkonstantine) [inline config/log4j.properties]: this variable (`ksql.log.dir`) doesn't seem to be used elsewhere in this file.

File: config/log4j-file.properties [hunk 2]
```diff
@@ -6,6 +6,7 @@ log4j.rootLogger=WARN, default.file
 log4j.appender.default.file=io.confluent.ksql.util.TimestampLogFileAppender
 log4j.appender.default.file.ImmediateFlush=true
 log4j.appender.default.file.append=false
-log4j.appender.default.file.file=${ksql.log.dir}/cli-%timestamp.log
+
+log4j.appender.default.file.file=${ksql.log.dir}/cli/cli-%timestamp.log
```
Comment (kkonstantine) [inline config/log4j-file.properties]: Should the directory be called `ksql` instead of `cli`?

File: bin/ksql-run-class [hunk 3]
```diff
@@ -3,20 +3,16 @@
...
+if [ "x$LOG_DIR" = "x" ]; then
+  LOG_DIR="$base_dir/logs"
 fi
 
 # create logs directory
-if [[ -n $LOG_DIR ]]; then
-    [[ -d $LOG_DIR ]] || mkdir -p "$LOG_DIR"
-#    KSQL_LOG4J_OPTS="-Dksql.log.dir=$LOG_DIR ${KSQL_LOG4J_OPTS}"
+if [ ! -d "$LOG_DIR" ]; then
+  mkdir -p "$LOG_DIR"
```
Comment (kkonstantine) [inline bin/ksql-run-class]: I can't add a comment inline, but consider adding `monitoring-interceptors` in the classpath below as: 

`for library in "confluent-common" "rest-utils" "ksql-engine" "ksql-rest-app" "ksql-cli" "ksql" "monitoring-interceptors"; do`

where now we have: 
`for library in "confluent-common" "rest-utils" "ksql-engine" "ksql-rest-app" "ksql-cli" "ksql"; do`

File: config/log4j-file.properties (hunk 2 above)
Comment (hjafarpour) [reply config/log4j-file.properties]: It's ksql-cli. I changed it to `ksql-cli`.

File: bin/ksql-run-class (hunk 3 above)
Comment (hjafarpour) [reply bin/ksql-run-class]: Done.

File: config/log4j-file.properties (hunk 2 above)
Comment (kkonstantine) [reply config/log4j-file.properties]: That would be my other suggestion. Cool, thx!

**Comment by hjafarpour** [comment]
@ewencp wanna take a look at this again?

File: bin/ksql [hunk 4]
```diff
@@ -4,11 +4,29 @@
 #
 # Use shellcheck to lint this file
 #
```
Comment (dguy) [inline bin/ksql]: probably should remove these comments as aren't using shellcheck anymore

**Pull request merged**